pkgs/desktops/gnome-3/extensions/emoji-selector/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, glib, gettext }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-emoji-selector";
+  version = "19";
+
+  src = fetchFromGitHub {
+    owner = "maoschanz";
+    repo = "emoji-selector-for-gnome";
+    rev = "${version}";
+    sha256 = "0x60pg5nl5d73av494dg29hyfml7fbf2d03wm053vx1q8a3pxbyb";
+  };
+
+  uuid = "emoji-selector@maestroschan.fr";
+
+  buildInputs = [ glib ];
+
+  buildPhase = ''
+    glib-compile-schemas ./${uuid}/schemas
+  '';
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions
+    cp -r ${uuid} $out/share/gnome-shell/extensions
+  '';
+
+  meta = with stdenv.lib; {
+    description =
+      "This GNOME shell extension provides a searchable popup menu displaying most emojis";
+    license = licenses.gpl3;
```

Comment:
Looks like `gpl3Plus`: https://github.com/maoschanz/emoji-selector-for-gnome/blob/f67019db7094c1fe2ecf71fcbc458a9ebf52c99b/emoji-selector%40maestroschan.fr/extension.js#L6-L9

Reply:
Does not appear resolved.

Reply:
Apologies, I was a bit aggressive with the `git add -p` . Fixed